trp_project_scrum/models/res_users.py

```diff
@@ -0,0 +1,22 @@
+from openerp import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @api.multi
+    @api.depends('groups_id')
+    def _is_collaborator(self):
+        for this in self:
+            collab_groups = [
+                x for x in this.sudo().groups_id.ids if x in
+                [
+                    self.sudo().env.ref('trp_project_scrum.collaborators').id,
+                    self.sudo().env.ref(
+                        'trp_project_scrum.collaborators_followers'
+                    ).id
+                ]
+            ]
+            this.is_collaborator = bool(collab_groups)
+    is_collaborator = fields.Boolean(
```

Comment:
i think this field can go away and in your code, check for has_group (this is ormcached)